RecoLocalCalo/HGCalRecProducers/interface/HGCalCLUEAlgo.h

```diff
@@ -48,13 +49,15 @@ class HGCalCLUEAlgo : public HGCalClusteringAlgoBase {
      nonAgedNoises_(ps.getParameter<edm::ParameterSet>("noises").getParameter<std::vector<double> >("values")),
      noiseMip_(ps.getParameter<edm::ParameterSet>("noiseMip").getParameter<double>("value")),
      initialized_(false),
-     points_(2*(maxlayer+1)),
-     minpos_(2*(maxlayer+1),{ {0.0f,0.0f} }),
-     maxpos_(2*(maxlayer+1),{ {0.0f,0.0f} }) {}
+    //  layerTiles_(2*(maxlayer+1)),
```

Comment:
When you will integrate the fix for the segmentation violation, please also manage to remove this now unneeded commented out line